tests/roles/backend_services/tasks/main.yaml

```diff
@@ -91,13 +97,22 @@
       args:
         chdir: "../config"
 
+- name: execute alternative tasks when source env is ODPdO
+  ansible.builtin.include_tasks: ospdo_backend_services.yaml
+  when: ospdo_src| bool
+  vars:
+    deploy_ctlplane_ospdo: true
+
 - name: Deploy the podified control plane
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc apply -f ../config/tmp/test_deployment.yaml
-  # args:
-  #   chdir: "../config"
+    mkdir -p tmp
```

Comment:
This needs to not change, currently it's reverting #592.

Reply:
right patching